docs/adr/adr-010-incentivized-testnet-monitoring.md

```diff
@@ -0,0 +1,286 @@
+
+### Which actor should run OTEL Collector(s) during the incentivized testnet?
+
+#### Scenario A: Node operators
+
+![scenario a](./img/incentivized-testnet-monitoring-scenario-a.png)
+
+Pros
+
+- This deployment architecture is more representative of mainnet where node operators will run their own telemetry stack to monitor their node. Exposing node operators to OTEL Collector during incentivized testnet allows them to practice this deployment architecture prior to mainnet.
+- Node operators will have an "incentive" to maintain high uptime for their OTEL Collector.
+
+Cons
+
+- Additional operational burden for incentivized testnet participants. We can mitigate this concern by providing easy install steps and scripts.
+
+#### Scenario B: Celestia team
+
+![scenario b](./img/incentivized-testnet-monitoring-scenario-b.png)
+
+Pros
+
+- It will be easier for nodes to participate if they only have to deploy one piece of software (celestia-node) and not two (celestia-node and OTEL Collector).
+
+Cons
+
+- Node operators will lose the ability to monitor their own celestia-node. Since opentelemetry-go supports configuring only one exporter ( [open-telemetry/opentelemetry-go#3055](https://github.com/open-telemetry/opentelemetry-go/issues/3055)), if node operators were obligated to export metrics to a Celestia team managed OTEL Collector endpoint, they wouldn't be able to export to their own OTEL Collector (and by proxy any telemetry platforms they wish to use). This violates the desideratum:
+
+    > We would like to make it possible for node operators to monitor their own nodes with existing telemetry tools (e.g. Grafana and Uptrace)
+
+- If the Celestia team took on this responsibility and failed to provide a highly available solution, then node operators would be penalized for downtime of a component they have no control over.
+- We expect 1500+ node operators during the incentivized testnet and there is minimal documentation on the scale of workload an individual OTEL Collector can handle. We'd have to design and operate a best-effort highly available OTEL Collector fleet to maintain high uptime for node operators. At this time no cloud managed offerings for OTEL Collector exist.
+
+#### Scenario C: Both. Node operators by default and Celestia team as a best-effort fallback
+
+![scenario c](./img/incentivized-testnet-monitoring-scenario-c.png)
+
+Pros
+
+- Optionality for node operators who don't want to deploy an OTEL Collector to rely on a best-effort OTEL Collector provided by Celestia team.
+
+Cons:
+
+- This option increases the cognitive load on node operators who now have an additional decision at deployment time.
+- Increased operational burden on Celestia team during incentivized testnet (and beyond).
+
```

Comment:
Slack feedback from @musalbas

> So I think the default should be that telemetry is sent to our collector by default (if they agree to the data collection), and nodes can optionally run their own telemetry with a separate binary if they want it

@rootulp to add an explicit **Scenario D** here

Reply:
Done in [7ffa115](https://github.com/celestiaorg/celestia-node/pull/922/commits/7ffa115337aa21e62177483e68c64763041cf615)